packages/retag/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "main": "dist/index.js",
   "bin": {
-    "retag": "./dist/retag.js"
+    "retag": "./dist/index.js"
```

Comment:
Wow, I was going to leave a comment about how weird it was that we had both main and bin for the same thing, but it actually checks if it's being invoked as main